### PR TITLE
Fix FEP number missing in FEP page header

### LINKED ISSUE #7: FEP number missing in FEP page header
On FEP page in header there is FEP title, but the number i missing.

See https://frostfs.info/proposals/proc/fep-0001/

### PR DESCRIPTION
**Pull request by corsairdnb**
Add FEP number in page title.

Closes: #7

### CODE REVIEW

File: layouts/partials/title.html [hunk 1]
```diff
@@ -0,0 +1 @@
+{{ with .Params.FEP }}FEP {{ . }} – {{ end }}{{ .Title }}
```
Comment (alexvanin) [inline layouts/partials/title.html:1]: By the way, do we have any policies of having `\n` in the end of files or not?

**Review by alexvanin — DISMISSED** [review]

File: layouts/partials/title.html (hunk 1 above)
Comment (realloc) [reply layouts/partials/title.html:1]: Having EOL at the end of file seems to be the universal standard. We should follow it unless we know exactly why we don't need EOL at some place.

**Comment by corsairdnb** [comment]
@realloc @alexvanin review please :)

**Review by realloc — APPROVED** [review]
Added theme submodule update to keep it consistent

**Review by alexvanin — APPROVED** [review]

**Pull request merged**